sqlite/test/general/strictMode.test.js

```diff
@@ -0,0 +1,105 @@
+const cds = require('../../../test/cds.js')
```

Comment:
This test should probably be inside `test/compliance`. That way it would also be included inside the other database services their tests.